src/matchers/toStartWith/index.js

```diff
@@ -0,0 +1,28 @@
+import { matcherHint, printExpected, printReceived } from 'jest-matcher-utils';
+
+import predicate from './predicate';
+
+const passMessage = (prefix, string) => () =>
```

Comment:
I think the error message would be better to show the expected too.

```js
const passMessage = (prefix, string) => () =>
  matcherHint('.not.toStartWith') +
  '\n\n' +
  'Expected string to not start with:\n' +
  `  ${printExpected(string)}`;
  'Received:\n' +
  `  ${printReceived(prefix)}`;
```

Reply:
Definitely - it's been added.